.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
```

Comment:
I haven't read the devcontainer spec. The aspnetcore one has a `build {..}` section, this does not?
https://github.com/dotnet/aspnetcore/blob/4eafb8ecdcf25b054b4bcda69ab1ecb498aece10/.devcontainer/devcontainer.json#L5